Modules/.AL-Go/settings.json

```diff
@@ -13,9 +13,10 @@
                         "System Tests"
     ],
     "artifact":  "bcinsider//21.5/base/latest/{INSIDERSASTOKEN}",
-    "enablePerTenantExtensionCop":  false,
```

Comment:
What about `enablePerTenantExtensionCop`? Why did it become `enableAppSourceCop`?

Reply:
Well enablePerTenantExtensionCop was never enabled - But let's try :)

Reply:
Build seems to work. Let's enable it